driver/configuration/provider_viper.go

```diff
@@ -305,7 +299,13 @@ func (v *ViperProvider) PipelineConfig(prefix, id string, override json.RawMessa
 		return errors.WithStack(result.Errors())
 	}
 
-	v.configCachge[hash] = config
+	marshalled, err = json.Marshal(config)
```

Comment:
`marshalled` should already be the proper type and payload (see https://github.com/ory/oathkeeper/pull/348/files/ede19d9b718fa4d91191ae9d138e89546305440c..8231b9aa87680589b8c42c25a46c5188acc446ca#diff-2449a6ea083767b149400840c05f41bdR296) so I don't think we need to encode it again!

Reply:
yeah :D quiet obviously... Removed the unnecessary marshall...